src/deepsparse/loggers/build_logger.py

```diff
@@ -263,11 +263,13 @@ def build_data_loggers(
                     metric_functions, loggers, identifier_prefix
                 )
             )
+            continue
```

Comment:
So that metric group (e.g. `func: image_classification`) does not get treated as a normal metric function.

Reply:
Would be a useful comment!